apps/console/src/styles/app.css

```
@@ -0,0 +1,313 @@
+    --shadow-drawer:    -16px 0 32px -16px rgba(0,0,0,0.40);
+  }
+}
+
+/* ── 2. Base reset ───────────────────────────────────────────────────── */
+*, *::before, *::after { box-sizing: border-box; }
+html, body { height: 100%; margin: 0; overflow: hidden; }
+
+body {
+  background: var(--app-bg);
+  color: var(--app-fg);
+  font-family: 'Manrope', system-ui, sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Manrope" (font-family-name-quotes)

(font-family-name-quotes)